src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
```

Comment:
When splitting the `sourceString`, it may be beneficial to also trim each declaration to handle cases where there might be leading or trailing whitespace around the semicolon.